xontrib/vox.py

```diff
@@ -1,77 +1,99 @@
 """Python virtual environment manager for xonsh."""
 
 import sys as _sys
+import argparse as _argparse
 import xontrib.voxapi as _voxapi
 
 class _VoxHandler:
     """Vox is a virtual environment manager for xonsh."""
+    parser = _argparse.ArgumentParser(prog='vox', description=__doc__)
```

Comment:
It is probably better to have the parser be constructed the first time it is used (maybe as a property?) rather than at import time. You can see some examples of cached parser construction through out xonsh. We want our imports to be as fast as reasonably possible.